### PR TITLE
Mariadb docs update

### PR DESCRIPTION
**Pull request by grooverdan**
MariaDB plc has joined maintenance of this image.

The logo was disproportionally large when added, so restored to consistent size.

The page was drasticly oversized so I've moved most of this to our KB documentation.

Did some fairly extensive editing too.

I'm expecting examples to be contributed soon so the docs are slightly ahead of reality.

### CODE REVIEW

File: mariadb/content.md [hunk 1]
```diff
@@ -102,79 +164,35 @@ $ docker run -it --rm %%IMAGE%%:latest --verbose --help
...
 
 # Caveats
 
 ## Where to Store Data
 
 Important note: There are several ways to store data used by applications that run in Docker containers. We encourage users of the `%%IMAGE%%` images to familiarize themselves with the options available, including:
 
--	Let Docker manage the storage of your database data [by writing the database files to disk on the host system using its own internal volume management](https://docs.docker.com/engine/tutorials/dockervolumes/#adding-a-data-volume). This is the default and is easy and fairly transparent to the user. The downside is that the files may be hard to locate for tools and applications that run directly on the host system, i.e. outside containers.
+-	Use a named container using the container manager to manage the storage of your database data [by writing the database files to disk on the host system using its own internal volume management](https://docs.docker.com/engine/tutorials/dockervolumes/#adding-a-data-volume). This is the default and is easy and fairly transparent to the user. The downside is that the files may be hard to locate for tools and applications that run directly on the host system, i.e. outside containers.
```
Comment (yosifkit) [inline mariadb/content.md:195]:
```suggestion
-	Use a named volume using the container manager to manage the storage of your database data [by writing the database files to disk on the host system using its own internal volume management](https://docs.docker.com/engine/tutorials/dockervolumes/#adding-a-data-volume). This is the default and is easy and fairly transparent to the user. The downside is that the files may be hard to locate for tools and applications that run directly on the host system, i.e. outside containers.
```

File: mariadb/content.md [hunk 2]
```diff
@@ -186,179 +204,37 @@ The Docker documentation is a good starting point for understanding the differen
 	$ docker run --name some-%%REPO%% -v /my/own/datadir:/var/lib/mysql -e MARIADB_ROOT_PASSWORD=my-secret-pw -d %%IMAGE%%:latest
 	```
 
-The `-v /my/own/datadir:/var/lib/mysql` part of the command mounts the `/my/own/datadir` directory from the underlying host system as `/var/lib/mysql` inside the container, where MariaDB by default will write its data files.
+The `-v /my/own/datadir:/var/lib/mysql:Z` part of the command mounts the `/my/own/datadir` directory from the underlying host system as `/var/lib/mysql` inside the container, where MariaDB by default will write its data files.
```
Comment (yosifkit) [inline mariadb/content.md:207]: Did you want the `:Z` on the `docker run` line on 204 as well?

**Review by yosifkit — COMMENTED** [review]
Sorry for the delay. I have just a couple small comments.

File: mariadb/content.md (hunk 2 above)
Comment (grooverdan) [reply mariadb/content.md:207]: sure

**Review by yosifkit — APPROVED** [review]

**Pull request merged**